SpaceShooter/SpaceShooter/Animator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Space_Shooter;
+
+namespace Space_Shooter
+{
+    internal class Animator
+    {
+        GridBuffer Buffer;
```

Comment:
```suggestion
        private readonly GridBuffer Buffer;
```

Fields should have their access levels explicitly set, and marked as readonly if they are set at initialisation and never changed.